esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3261,39 +3169,32 @@ bool CFileSprayEx::onDropZoneFiles(IEspContext &context, IEspDropZoneFilesReques
         if (dropZoneList.ordinality())
             resp.setDropZones(dropZoneList);
 
-        if (netAddressStr.length() < 1)
-            return true;
-
-        char pathSep = '/';
-        if (osStr && *osStr)
-        {
-            int os = atoi(osStr);
-            if (os == OS_WINDOWS)
-                pathSep = '\\';
-        }
+        const char* dzName = req.getDropZoneName();
```

Comment:
It is optional (only used for new environment). For old environment, checkDropZoneIPAndPath() returns true if a dropzone matches IP and path. If blank, a null is given to getDropZoneFiles() and  checkDropZoneIPAndPath().